post_installation_configuration/ibmz-post-install.adoc

```diff
@@ -36,8 +36,10 @@ include::modules/ibmz-configure-devices-manually.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-See link:https://www.ibm.com/docs/en/linux-on-systems?topic=linuxonibm/com.ibm.linux.z.ludd/ludd_c_perscfg.html[Persistent device configuration] in {ibm-name} Documentation.
 
+* link:https://www.ibm.com/docs/en/linux-on-systems?topic=commands-chzdevl[chzdev - Configure {ibm-z-name} devices] in {ibm-name} Documentation.
```

Comment:
Link does not bring me to a direct doc location but instead:

![Screenshot from 2024-07-02 17-27-18](https://github.com/openshift/openshift-docs/assets/57954076/cc37ed70-3eb4-4354-96c6-c608b9376e87)

Maybe this was intended?

Link style suggestion:

> [chzdev - Configure {ibm-z-name} devices] ({ibm-name} Documentation).

Matches [this rule](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute).